models/staging/stg_customers.sql

```diff
@@ -0,0 +1,5 @@
+with source as (
+    select * from {{ source('dbt_christine', 'customers') }}
+)
+
```

Comment:
best to rename this following the style guide naming conventions (as well as the rest of the staging tables)

Reply:
Renamed columns in stg models and tested:
<img width="762" alt="Screen Shot 2020-04-05 at 12 19 26 AM" src="https://user-images.githubusercontent.com/60892553/78467507-33f39580-76d3-11ea-9269-2b548652426c.png">